Controller/Payment/Webhook.php

```diff
@@ -237,9 +273,17 @@ protected function authorize(array $post)
                 {
                     $payment = $order->getPayment();
 
-                    if (($order->getStatus() === static::STATUS_PENDING) and
-                        (empty($payment->getLastTransId()) === true) )
+                    if ($order->getState() === static::STATE_NEW and
+                        ($order->getStatus() === static::STATUS_CANCELED or
+                        $order->getStatus() === static::STATUS_PENDING)
+                    )
                     {
+
+                        $this->logger->info("Razorpay Webhook: "
+                        . " Event: " . $post['event']
```

Comment:
Please follow the multiple arguments, alignment formatting rules

Reply:
@nikhilsharma1100 Updated alignment formatting for the multiple arguments